modules/dxp/apps/commerce/commerce-machine-learning-test/src/testIntegration/java/com/liferay/commerce/machine/learning/recommendation/test/ProductContentCommerceMLRecommendationManagerTest.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.commerce.machine.learning.recommendation.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.commerce.machine.learning.recommendation.ProductContentCommerceMLRecommendation;
+import com.liferay.commerce.machine.learning.recommendation.ProductContentCommerceMLRecommendationManager;
+import com.liferay.portal.kernel.model.Company;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.util.CompanyTestUtil;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.search.test.util.IdempotentRetryAssert;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Riccardo Ferrari
+ */
+@RunWith(Arquillian.class)
+public class ProductContentCommerceMLRecommendationManagerTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		_company = CompanyTestUtil.addCompany();
+
+		_productContentCommerceMLRecommendations = _populateEntries(4, 11);
+	}
+
+	@Test
+	public void testGetProductContentCommerceMLRecommendations()
+		throws Exception {
+
+		Stream<ProductContentCommerceMLRecommendation>
+			productContentCommerceMLRecommendationStream =
+				_productContentCommerceMLRecommendations.stream();
+
+		List<ProductContentCommerceMLRecommendation>
+			expectedProductContentCommerceMLRecommendationList =
```

Comment:
@riccardo-ferrari we should name this `expectedProductContentCommerceMLRecommendations`

Reply:
Done thanks